apis/python/src/tiledbsoma/_query.py

```diff
@@ -457,13 +464,21 @@ def to_anndata(
         var_joinids = self.var_joinids()
 
         x_matrices = {
-            _xname: tp.submit(
-                _read_as_csr,
-                layer,
-                obs_joinids,
-                var_joinids,
-                self._indexer.by_obs,
-                self._indexer.by_var,
+            _xname: (
```

Comment:
why are only X/layer matrices loaded with Dask-backed arrays?  I.e, why not obsm/varm/obsp/varp?   (AnnData limitation?)

Reply:
I think this is a roadmap question, eg., for Aaron.  I'm fine with whatever you guys decide